server/src/main/java/org/hyperskill/community/flashcards/card/ExampleDataInitializer.java

```diff
@@ -101,18 +102,12 @@ public void run(String... args) {
         var collectionNames = mongoTemplate.getDb().listCollectionNames();
         for (var collectionName : collectionNames) {
             log.info("Collection: {}", collectionName);
-            getAllCards(collectionName).stream().map(card -> {
-                        if (card instanceof QuestionAndAnswerCard qac) {
-                            return qac;
-                        } else if (card instanceof SingleChoiceQuiz scc) {
-                            return scc;
-                        } else if (card instanceof MultipleChoiceQuiz mcc) {
-                            return mcc;
-                        } else {
-                            throw new IllegalStateException("Unknown card class " + card.getClass().getName());
-                        }
+            getAllCards(collectionName).stream()
+                    .map(card -> switch (card) {
+                        case SingleChoiceQuiz scq -> scq;
+                        case MultipleChoiceQuiz mcq -> mcq;
+                        case QuestionAndAnswerCard qac -> qac;
                     })
-                    .map(Objects::toString)
                     .forEach(System.out::println);
         }
```

Comment:
still: whatis the purpose of the map() here?
Why not just:
getAllCards(collectionName).foreach(c -> log.debug("" + c); // or log.info if you prefer for now

Reply:
To prove that cards are successfully cast to their class. It doesn't make sense to have tests at this stage, so it's a sort of substitution for them. Will remove the unnecessary queries and log entries in the final commit.